Marlin/ultralcd.cpp

```diff
@@ -1260,14 +1260,25 @@ void kill_screen(const char* lcd_msg) {
 
   #if ENABLED(DELTA_CALIBRATION_MENU)
 
+    static void _goto_tower_pos(const float &a) {
+      char cmd[20];
+      sprintf_P(cmd, PSTR("G1 X%i Y%i"), int(-(DELTA_PRINTABLE_RADIUS) * sin(a)), int((DELTA_PRINTABLE_RADIUS) * cos(a)));
+      enqueue_and_echo_commands_P(PSTR("G1 F8000 Z4\nG1 F4000"));
+      enqueue_and_echo_command(cmd);
+    }
```

Comment:
`Z_HOMING_HEIGHT` on a Delta is meaningless. The default is off, or zero. It's not the best choice. This is for calibration, where you want the nozzle very close to the bed. I would opt for 2mm, but 4mm is almost certainly "safe" even when the delta geometry is pretty far off. I'd rather we "dictate" that this is 4mm.